test/export/test_serialize.py

```diff
@@ -103,7 +103,7 @@ def forward(self, x):
         input.requires_grad = True
         exported_module = export(MyModule(), (input,))
 
-        serialized, _ = ExportedProgramSerializer().serialize(exported_module)
+        serialized, *_ = ExportedProgramSerializer().serialize(exported_module)
```

Comment:
I don't quite like this *_  because it makes it look like there are a variable number of remaining things. IMO at this point we should consider using a dataclass / named tuple and just extract from it (`result.serialized`).